style-guide/blog-guidelines.rst

```diff
@@ -0,0 +1,181 @@
+- Does the blogs describe something that Rackspace does better than the
+  competition or that no other competitor offers? Does the _blog provide
+  quantifiable and defensible information (such as performance metrics)
+  showing the Rackspace edge over the competition? If yes, explain.
+
+
+Blog writing suggestions
+------------------------
+
+When you write your content, we recommend the following rules of thumb:
+
+-  Length: Rackspace blog posts generally run between 500-1,000 words, but let
+   the content be your guide. Take the space you need to tell your story, but
+   don’t pad it unnecessarily. Going long is okay, too, as long as it’s worth
+   reading.
+
+-  Tools: We recommend using a word processor, like Microsoft Word, or a text
+   editor for your writing.
+
+-  General format:
+
+   - An attention-grabbing opening, known in the news business as a “lede”
+
+   - A “nut graf” - a sentence or two letting readers know what you’re going
+     to be talking about and why it matters
+
+   - Some background information or context, such as the history of a product
+
+   - A conclusion (and a call to action, if appropriate) with links and contact
+     information
+
+Be consistent in style with other blogs on the Rackspace Developer Blog site.
+
+Bullet points are useful for listing features.
+
+Think “be helpful” rather than trying to sell a product or service. WHAT does
+the product do? HOW does it help the customer?
+
+-  Write a good lede: When crafting your lede, consider how the information
+   in your post helps customers. An announcement of a new product is not a
+   lede, but that product’s money-saving capacity may well be one! For example:
+
+   Not good: “Today Rackspace is pleased to announce the release of four new
+   features in Office 365.”
+
+   Good: “Increased storage, security, and cost-saving measures are now
+   available to enhance the already-robust Office 365 suite available from
+   Rackspace, making it even more attractive to a wider variety of business
+   users.”
+
+-  Stay focused on your topic: If you find yourself straying, you may have
+   enough material for a series. It’s okay to break up a big topic into more
+   digestible bites. Formulas like How to, Top 5 or 10, and curated lists tend
+   to do well.
+
+-  Show don’t tell. Can you create a graphic or image, or add a screen shot to
+   illustrate your point?
+
+-  Avoid jargon! Rackers (and the entire tech industry) tend to speak in
+   jargon. Unless your post is aimed at a technical audience, use plain
+   English. There is never a reason to use business jargon.
+
+-  Link link link: A first product mention? Link to the product page. Outside
+   accolades? Link to it. Citing a study or source or article? Link to it.
+   Have we written about it before? Link to it. Mention a partner or customer?
+   Link to their site.
+
+-  Keep it concise: The Internet have shrunken everyone’s attention span.
```

Comment:
Sorry, should have been more clear in the last comment, "have" should also be "has"

Reply:
Add here to that image should contribute to content of the blog post.

Reply:
fixed